torchaudio/functional/filtering.py

```diff
@@ -7,6 +8,23 @@
 import torchaudio._internal.fft
 
 
+def lfilter_core_generic_loop(input_signal_windows: Tensor, a_coeffs_flipped: Tensor, padded_output_waveform: Tensor):
```

Comment:
Can you prefix this with underscore `_` and move the definition and the following `lfilter_core_cpu_loop ` assignment closer to `def lfilter`?

Also please prefix `lfilter_core_cpu_loop` with underscore `_` as well.